.github/workflows/pull-request-ci.yml

```
@@ -53,7 +53,7 @@ jobs:
         uses: actions/cache@v2
         with:
           path: external
-          key: ${{ runner.os }}-external-${{ hashFiles('src/Radium-Engine/external/**/CMakeLists.txt') }}
+          key: ${{ runner.os }}-external-v1-${{ hashFiles('src/Radium-Engine/external/**/CMakeLists.txt') }}
```

Well, we don't need this, I think, but rather have something that depends on compiles options. maybe the whole config ?
For instance (not tested) 
```suggestion
          key: ${{ matrix.config }}-external-${{ hashFiles('src/Radium-Engine/external/**/CMakeLists.txt') }}
```
But I'm not sure the matrix.config will convert to a string suitable for key definition.
The hashFiles ensure that modification to the externals CMakeLists.txt create a new key,
we can also use all Radium's CMakeLists, but might be overkill. 

I'm also fine with the version, but it make a manual update needed here ...